.circleci/config.yml

```
@@ -53,6 +53,36 @@ commands:
     description: Setup android Google services config
     steps:
       - run: echo $ANDROID_GOOGLE_SERVICES_JSON > android/app/google-services.json
+  setup_staging_secrets:
+    description: Setup staging secrets env variables
+    steps:
+      - run:
+          name: Setup secrets env variables
+          command: |
+            echo "CODEPUSH_KEY_ANDROID=$CODEPUSH_KEY_ANDROID_STAGING" >> .env.staging
```

```suggestion
            export CODEPUSH_KEY_ANDROID=$CODEPUSH_KEY_ANDROID_STAGING
```

pk ne pas faire ça ? comme ça évite de modifier .env.staging et de devoir checker `git status -s --assume-unchanged ../.env.*`

---

parce que ça ne modifie que pour la CI, c'est pas dans les variables d'env du projet en soit

---

de quoi?